### DISCUSSION TITLE
`FUNDING.yml` should be removed on repo creation from template

### DISCUSSION THREAD

**Discussion by ahmedali8** [paulrberg/foundry-template#16] [Ideas]
Hi @PaulRBerg

Similar to #13, imo `FUNDING.yml` should also be removed when a user creates a repo using the `Use this template`. What is your opinion?

**Comment by PaulRBerg** [comment]
Oh lol, we should absolutely remove that file, yes. Could you make a PR for this, please?

**Reply by ahmedali8** [reply]
Sure, I'll create a PR for this.

**Reply by PaulRBerg** [reply]
@ahmedali8 - don't worry anymore, I have just pushed a commit to fix this:

https://github.com/PaulRBerg/foundry-template/commit/86208cef2181076dbb36479b7efb8c307380b344
